internal/engine/eval/rego/result.go

```diff
@@ -152,7 +154,7 @@ func (*denyByDefaultEvaluator) parseResult(rs rego.ResultSet, entity protoreflec
 	}
 
 	entityName := getEntityName(entity)
-	return engerrors.NewDetailedErrEvaluationFailed(
+	return nil, engerrors.NewDetailedErrEvaluationFailed(
```

Comment:
Maybe just a TODO, but should we return message and entity name as an evaluation result in the future?

(We may want to think about the format here, I'm not sure)

Reply:
This was a requirement for improved [evaluation details story](https://github.com/stacklok/minder-stories/issues/43), but the request was discussed via Slack IIRC.

Reply:
I agree, I wanted to start with the constraints engine because that's what the Trusty rule will use, but the deny-by-default engine will be updated too.